docs/contracts/burner.md

```diff
@@ -78,12 +78,20 @@ Returns the total non-cover shares ever burnt.
 function getNonCoverSharesBurnt() external view returns (uint256)
 ```
 
-### function getExcessStETH
+### Function: getExcessStETH
```

Comment:
Maybe, should do it consistently? In other docs we have `## Methods` and just `functionname()` with parenthesis without `Function:` prefix

Reply:
Indeed its only in Burner, thanks, fixed